skills/cve-fix/Makefile

```
@@ -0,0 +1,39 @@
+	@echo ""
+	@echo "  Repos: $(shell grep '^\s*- name:' repos.yaml | awk '{printf "%s ", $$NF}')"
+	@echo ""
+	@echo "  Config: repos.yaml"
+	@echo ""
+
+fix:
+	$(if $(REPO),,$(error REPO is required (name from repos.yaml or absolute path)))
+	$(if $(BRANCH),,$(error BRANCH is required (e.g., 0.23, release-0.23, or devel)))
+	./scripts/fix-all.sh "$(REPO)" "$(BRANCH)"
+
+all:
```

checkmake (0.3.2)

[warning] Target body for "all" exceeds allowed length of 5 lines (6).

(maxbodylength)